handbook/brand.md

```diff
@@ -393,6 +393,119 @@ When conducting an incident post-mortem, answer the following three questions:
 2. Root Cause: Why did this error happen?
 3. Side effects: did this error have any side effects? e.g., did it corrupt any data? Did code that was supposed to run afterward and “finish something up” not run, and did it leave anything in the database or other systems in a broken state requiring repair? This typically involves checking the line in the source code that threw the error.
 
+# Article formatting guidelines
+
+To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in <ins><strong>Markdown</strong></ins>.
+
+### On this page
+- [Layout](#heading=h.7116hdoxal2s)
+- [Images & screenshots](#heading=h.f2zdns6kprrh)
+- [Meta tags](#heading=h.4ayd6zwgnyo1)
+- [Customizable CTA](#heading=h.2nnazmktmx29)
+- [Other pages of interest](#heading=h.3ii71mq08q0d)
+
+## Layout
+The following layout guide aims to help you create consistently formatted articles. For an existing article example, check out the [Markdown](https://raw.githubusercontent.com/fleetdm/fleet/main/articles/tales-from-fleet-security-speeding-up-macos-updates-with-nudge.md) and the [finished result](https://fleetdm.com/securing/tales-from-fleet-security-speeding-up-macos-updates-with-nudge).
+
+### Hero image
+Consider adding a hero image for a more significant impact. Get in touch with Digital Experience via #content on Slack to make a request. 
+
+### Table of contents
+For long articles or guides, consider adding a table of contents.
+
+### Introduction
+It’s good practice to start your article with a clear summary of what you will be discussing.
+
+### Main content
+The main body of your article.
+
+### Conclusion
+It’s a good idea to finish your article with a clear closing statement.
+
+### Add a customizable CTA
+Add a CTA at the end of your article. See [Customizable CTA](#heading=h.2nnazmktmx29) below for instructions on creating a CTA tailored to your article topic.
+
+## Images & screenshots
+Images are a great way to help engage your readers. But consider the following before including images or screenshots in your article:
+
+- Does the image add value?
+- Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.
+
+## Meta tags
+These tags help … summary of what these are for (todo)
```

Comment:
Should "…" be filled in with something else?

Reply:
Yes, please, can you grab a moment with @eashaw and ask him for a summary of what the Meta tags do?

Reply:
This is on the schedule.